src/analysis/seminorm.lean

```diff
@@ -348,6 +389,21 @@ begin
   ...    < r   : by rwa mem_ball_zero at hy,
 end
 
+lemma ball_finset_sup_eq_Inter (p : ι → seminorm 𝕜 E) (s : finset ι) (e : E) {r : ℝ} (hr : 0 < r) :
```

Comment:
This lemma is redundant with the next. Could remove it and rename the nex `ball_finset_sup`? Also, you're missing `ball_sup` the binary version.

Reply:
Nice point about the binary version. I guess we should have a statement about the ball on the zero seminorm too?

Reply:
I think it's worth keeping both versions because `finset.inf` isn't a particularly natural way to speak about sets, as it uses `⊓`  instead of `∩`

Reply:
I've added the binary version, but I disagree with removing `ball_finset_sup_eq_Inter`. I did not find a trivial way to rewrite `s.inf (λ i, f i)` to `⋂ (i ∈ s), f i`, where `f : ι → set E` and neither did the library search, so I think it is ok to have both (and the version with intersections is more useful in my opinion).